### DISCUSSION TITLE
Complete Chinese and English documents advanced_guides/data_flow.md 完成中英文文档 advanced_guides/data_flow.md

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#549] [MMagic]
# Complete Chinese and English documents advanced_guides/data_flow.md 
 完成中英文文档 advanced_guides/data_flow.md 

Complete Chinese and English documents [https://github.com/open-mmlab/mmagic/blob/main/docs/en/advanced_guides/data_flow.md](https://github.com/open-mmlab/mmagic/blob/main/docs/en/advanced_guides/data_flow.md) 
and [https://github.com/open-mmlab/mmagic/blob/main/docs/zh_cn/advanced_guides/data_flow.md](https://github.com/open-mmlab/mmagic/blob/main/docs/zh_cn/advanced_guides/data_flow.md) 

完成中英文文档 [https://github.com/open-mmlab/mmagic/blob/main/docs/en/advanced_guides/data_flow.md](https://github.com/open-mmlab/mmagic/blob/main/docs/en/advanced_guides/data_flow.md) 
以及 [https://github.com/open-mmlab/mmagic/blob/main/docs/zh_cn/advanced_guides/data_flow.md](https://github.com/open-mmlab/mmagic/blob/main/docs/zh_cn/advanced_guides/data_flow.md) 

## Task Information

**Repository**：MMagic

**Difficulty**：Easy

**Points**：20

**Estimated Time Consumption**：7 days

## Task Status

**Claimant's Github ID**：aptsunny

**Claim Time**：2023年07月24日

**PR Link**：https://github.com/open-mmlab/mmagic/pull/1952

## Points to Note

❗️请注意：此任务属于超级视客营任务，不能被领取！请登陆 [超级视客营平台](https://openmmlab.com/activity/codecamp) 按照流程申请任务，我们会尽快审核～

**Comment by aptsunny** [comment]
https://github.com/open-mmlab/mmagic/pull/1952
